contracts/bridge/Bridge.sol

```diff
@@ -46,37 +46,42 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
 
     /**
      * Event to log the withdrawal of a token from the Bridge.
-     * @param kind The type of token withdrawn (KLAY/TOKEN/NFT).
+     * @param kind The type of token withdrawn (KLAY/ERC20/ERC721).
      * @param from is the requester of the request value transfer event.
-     * @param contractAddress Address of token contract the token belong to.
-     * @param amount is the amount for KLAY/TOKEN and the NFT ID for NFT.
+     * @param to is the receiver of the value.
+     * @param tokenAddress Address of token contract the token belong to.
+     * @param valueOrTokenId is the value of KLAY/ERC20 or token ID of ERC721.
      * @param requestNonce is the order number of the request value transfer.
      * @param uri is uri of ERC721 token.
      */
-    event RequestValueTransfer(TokenKind kind,
+    event RequestValueTransfer(
+        TokenKind kind,
```

Comment:
what about using TokenType type instead of kind?

Reply:
Done. Thanks. :)